webapp/src/Controller/Jury/ClarificationController.php

```diff
@@ -239,6 +257,7 @@ public function composeClarificationAction(
         #[MapQueryParameter]
         ?string $teamto = null,
     ): Response {
+        $this->warnClarificationBeforeContestStart();
```

Comment:
in addition, I was thinking whether we should add a modal dialog on submission warning of the implications